pkg/flowkit/output/prompt.go

```diff
@@ -157,6 +157,21 @@ func namePrompt() string {
 	return name
 }
 
+func secureNetworkKeyPrompt() string {
+	networkKeyPrompt := promptui.Prompt{
+		Label: "Enter host network key",
+		Validate: func(s string) error {
+			return nil
```

Comment:
Can we make sure it is a valid network key here? (attempt to parse as a hex-encoded ECDSA public key)

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/0e29e14541a6f2d948b2357a9b7ff9df72b5330b